packages/app/src/cli/services/app-logs/write-app-logs.test.ts

```diff
@@ -0,0 +1,66 @@
+import {writeAppLogsToFile} from './write-app-logs.js'
+import {AppEventData} from './poll-app-logs.js'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {writeLog} from '@shopify/cli-kit/node/logs'
+import {describe, expect, test, vi, beforeEach} from 'vitest'
+
+vi.mock('@shopify/cli-kit/node/logs')
+
+const DUMMY_JSON = {
+  someJson: 'someJSOn',
+}
+
+const APP_LOG: AppEventData = {
+  shop_id: 1,
+  api_client_id: 2,
+  payload: JSON.stringify(DUMMY_JSON),
```

Comment:
Nit: Since `DUMMY_JSON` is small and not used by anything else, maybe we could just inline it?

```suggestion
  payload: '{}',
```